common/lib/xmodule/xmodule/course_module.py

```diff
@@ -390,44 +390,46 @@ class CourseFields(object):
+                        "action_url": "CourseUpdates",
+                        "action_text": _("View Course Updates"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Set Important Dates for Your Course"),
+                        "short_description": _("Direct learners to edX DemoX"),
                         "long_description": _(
-                            "Establish your course's student enrollment and launch dates on the Schedule and Details "
-                            "page."
+                            "Provide a link to edX DemoX course (an introduction to the edX platform for new learners) in the "
+                            "first week's course content."
                         ),
                         "is_checked": False,
-                        "action_url": "SettingsDetails",
-                        "action_text": _("Edit Course Details &amp; Schedule"),
-                        "action_external": False,
+                        "action_url": "https://www.edx.org/course/demox-edx-demox-1",
+                        "action_text": "Go to edX DemoX",
+                        "action_external": True,
                     },
                     {
-                        "short_description": _("Draft Your Course's Grading Policy"),
+                        "short_description": _("Prompt learners to get started on course"),
                         "long_description": _(
-                            "Set up your assignment types and grading policy even if you haven't created all your "
-                            "assignments."
+                            "Provide a prompt to the learner on the course info page on how to get started with the course "
+                            "(e.g. \"To get started, click on the \"Courseware\" tab at the top of the page.\")"
                         ),
                         "is_checked": False,
-                        "action_url": "SettingsGrading",
-                        "action_text": _("Edit Grading Settings"),
+                        "action_url": "CourseUpdates",
+                        "action_text": _("View Course Updates"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Explore the Other Studio Checklists"),
+                        "short_description": _("Tell learners how to get help"),
                         "long_description": _(
-                            "Discover other available course authoring tools, and find help when you need it."
+                            "Explanation posted of how to get help with learner issues:"
+                            "(e.g. Course content issues should be posted in the Dicussions Forum, edX technical issues"
+                            "directed to Student Support Team via Help button)"
```

Comment:
Provide your learners with information about ways to get help. For example, on the Course Info page include \"To get help with the course, click the Discussion tab and post a question. To get help with a technical problem, click Help to send a message to edX Student Support.\"